src/stateFromMarkdown.js

```diff
@@ -1,11 +1,25 @@
 /* @flow */
 
-import MarkdownParser from './MarkdownParser';
 import {stateFromElement} from 'draft-js-import-element';
+import remark from 'remark';
+import html from 'remark-html';
+import {jsdom} from 'jsdom';
```

Comment:
Moved to browser parser module from Your link. Currently i'm having some difficulties with blockquote test not passing, will need to look whats the issue